apps/www/data/Pricing.json

```diff
@@ -379,5 +379,35 @@
         }
       }
     ]
+  },
+  "realtime": {
+    "title": "Realtime",
+    "icon": "M18.87,12.07l-6,9.47A1,1,0,0,1,12,22a.9.9,0,0,1-.28,0A1,1,0,0,1,11,21V15H6.82a2,2,0,0,1-1.69-3.07l6-9.47A1,1,0,0,1,12.28,2,1,1,0,0,1,13,3V9h4.18a2,2,0,0,1,1.69,3.07Z",
```

Comment:
@MildTomato i just copy/pasted this icon from somewhere. can you help me create an icon for realtime?

Reply:
we have re-made icons for all the products. so it will get replaced anyway